blockchain/blockchain.go

```diff
@@ -54,10 +54,18 @@ type Blockchain struct {
 type Verifier interface {
 	VerifyHeader(parent, header *types.Header) error
 	GetBlockCreator(header *types.Header) (types.Address, error)
+	PreStateCommit(header *types.Header, txn *state.Transition) error
 }
 
 type Executor interface {
-	ProcessBlock(parentRoot types.Hash, block *types.Block, blockCreator types.Address) (*state.BlockResult, error)
+	//ProcessBlock(parentRoot types.Hash, block *types.Block, blockCreator types.Address) (*state.BlockResult, error)
```

Comment:
Nitpick warning: put a space between the slashes and text in comments

Reply:
Thanks. I noticed it was leftover code. I removed at 464cea9821e28327cb3c8e85cda47c21753feec5